### PR TITLE
Retry initial connection, won't retry after disconnect

### PR DESCRIPTION
**Pull request by jonasfj**
I think this is better than nothing... Perhaps it's too many retries I don't..  Sometimes we need to retry a few times to get different DNS results...

### CODE REVIEW

**Comment by jonasfj** [comment]
See https://bugzilla.mozilla.org/show_bug.cgi?id=1297560

Please close the bug both lands...

File: src/exchanges.js [hunk 1]
```diff
@@ -383,30 +383,43 @@ Exchanges.prototype.connect = function(options) {
...
+      if (retry++ < 12) {
+        continue; // try again
+      }
+      throw err;
+    }
+    break;
+  }
+
+  // Create confirm publish channel
+  var channel = await conn.createConfirmChannel();
```
Comment (imbstack) [inline src/exchanges.js]: Defining this below where it is first used is a bit confusing to me.

Comment (imbstack) [inline src/exchanges.js]: Oh wait, I'm wrong with how this flow goes! Deleting.

**Comment by imbstack** [comment]
r+

ship-it!

**Pull request merged**